### DISCUSSION TITLE
请问打包下载能否调用下载器

### DISCUSSION THREAD

**Discussion by KanoYL** [AlistGo/alist#858] [Q&A] [answered]
文件夹或多数文件选择打包下载后，会直接使用浏览器下载
由于不同文件下载速度不同，所以想调用下载器多线程下载
如果复制链接到下载器（IDM、Aria2等），会使链接失效
复制直链或进入到文件直接下载可以正常调用下载器
请问能否有方法打包后不进行下载而输出直链以使用下载器下载？
谢谢

**Comment by xhofe** [comment, marked as answer]
无法调用，打包下载是前端实现，利用的是浏览器的[Streams_API](https://developer.mozilla.org/zh-CN/docs/Web/API/Streams_API)。

**Reply by KanoYL** [reply]
谢谢
